pyvrp/cpp/ProblemData.h

```diff
@@ -321,6 +323,9 @@ class ProblemData
      *     tw_late: int = np.iinfo(np.int64).max,
      *     max_duration: int = np.iinfo(np.int64).max,
      *     max_distance: int = np.iinfo(np.int64).max,
+     *     unit_distance_cost: int = 1,
+     *     unit_duration_cost: int = 0,
```

Comment:
I'm not a huge fan of these names, but at least it seems clear. Any suggestions for improvement are welcome :).